blur-no-alt.php

```diff
@@ -90,22 +90,22 @@ public function blur_no_alt_message_display_page_init() {
 
 		add_settings_section(
 			'blur_no_alt_front_end_blur_setting_section',
-			__( 'Front-end Blur Settings', 'blur-no-alt' ),
```

Comment:
Why take this out?

Reply:
This was an addition not a subtraction, so I think maybe this is confusion due to the opaque diffs (my fault on the file name change). I'll try to get that done in a couple hours so it's easier to review.